### ISSUE TITLE
Data Analysis 

### ISSUE THREAD

**Issue by varunbansal313** [professor-greebie/SENG8080-3-field_project#119]
Starting the data analysis of the dataset.

**Comment by varunbansal313** [comment]
I see there are 53 columns in the dataset, I think it will make more sense if we drop at least half the columns and then analyze the data, given the time constraint.

![image](https://github.com/professor-greebie/SENG8080-3-field_project/assets/97469778/9f3c114c-2f41-4e6e-9551-61e0d922af06)

**Comment by DelnaMathew** [comment]
I believe removing columns is part of data cleaning, typically handled by the data quality team. However, I would like to confirm if we plan to start the data analysis process by temporarily removing the columns.

**Comment by AniketTiwari19** [comment]
Yes I agree with @varunbansal313 and @DelnaMathew  dropping columns from a dataset for the initial analysis and confirm with the data quality team as it can be considered a useful approach because we are dealing with large dataset and doing this steps can save our time and also reduce complexity.

**Comment by Ashish8128** [comment]
Did we got any updates from the data quality team as they told that they will update us by Monday for the clean data?

**Comment by maulikpatel30** [comment]
They are still cleaning the data and will update shortly.

**Comment by svitolopes** [comment]
Moving on with the data visualization part, and thinking that with this dataset, we will not be able to continue with the issues and KPIs we discussed before. I created this list of questions that we can answer using the dataset, we will need to evaluate if, with the cleaned dataset we will continue to be able to answer these questions and choose which ones we will use.
1.	How many individuals were included in the study?
2.	What is the age distribution of patients in the dataset?
3.	What are the different cancer types present in the dataset, and how many samples are available for each type?
4.	What are the detailed subtypes for each cancer type?
5.	What is the distribution of patients based on race and sex?
6.	What are the different types of chemotherapy treatments used in the study?
7.	How many patients received cytotoxic chemotherapy treatment?
8.	What is the distribution of patients based on their smoking status?
9.	How long was the time from diagnosis to the start of treatment for each patient?
10.	What is the status of treatment for the patients?
11.	How many patients received radiation therapy as part of their treatment?
12.	What are the clonal hematopoiesis mutations identified in the blood samples?
13.	How many patients had clonal hematopoiesis mutations?
14.	Are there any correlations between the presence of clonal hematopoiesis mutations and specific cancer types?
15.	How does the distribution of chemotherapy treatments differ among cancer types?
16.	Is there any correlation between patient age and cancer type?
17.	How many samples were analyzed per patient on average?
18.	Are there any differences in treatment patterns based on the cancer type?
